templates/rhsm-katello-reconfigure.erb

```diff
@@ -71,9 +71,14 @@ if [ -d $CA_TRUST_ANCHORS ]; then
   cp $KATELLO_CERT_DIR/$KATELLO_CERT $CA_TRUST_ANCHORS
   update-ca-trust
 
-  #restart if docker service is installed
-  service docker status >/dev/null && \
+  # restart docker if it is installed and running
+  if [ -f /usr/lib/systemd/system/docker.service ]; then
```

Comment:
Unit files installed by packages are placed in `/usr/lib/systemd/system/`, but system administrators may also install unit files in `/etc/systemd/system/`. An administrator might even do something crazy like nuke the default service file and install a custom one to `/etc/systemd/system/`. As a result, this sort of test may be better:

    if which systemctl >&/dev/null; then echo true; else echo false; fi